packages/core/src/components/non-ideal-state/nonIdealState.tsx

```diff
@@ -94,13 +102,13 @@ export class NonIdealState extends AbstractPureComponent<NonIdealStateProps> {
     }
 
     private maybeRenderVisual() {
-        const { icon, iconSize } = this.props;
+        const { icon, iconSize, mutedIcon } = this.props;
         if (icon == null) {
             return undefined;
         } else {
             return (
                 <div
-                    className={Classes.NON_IDEAL_STATE_VISUAL}
+                    className={classNames(mutedIcon && Classes.NON_IDEAL_STATE_VISUAL)}
```

Comment:
Let's not remove the class in this PR, as I think it is still semantically valuable to be able to target the "visual" element in the DOM here. Instead, let's adjust the styles to target a new class applied to the `<Icon>` element below.

Reply:
```suggestion
                    className={Classes.NON_IDEAL_STATE_VISUAL}
```